ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -98,6 +98,12 @@
     reason="Tests are not running on AWS deployed cluster"
 )
 
+rh_internal_lab_only = pytest.mark.skipif(
```

Comment:
Small nitpick but our other similar marks all have the term `required`, can we keep that convention here and name this `rh_internal_lab_required`?